# contracts/ERC20Confidential/ERC20Confidential.sol  ->  contracts/ERC20Confidential/ERC20Confidential.fsol
# upstream 5138cb8 .. ported 4cefd22

diff --git a/contracts/ERC20Confidential/ERC20Confidential.sol b/contracts/ERC20Confidential/ERC20Confidential.fsol
index 5117ff2..1ef3217 100644
--- a/contracts/ERC20Confidential/ERC20Confidential.sol
+++ b/contracts/ERC20Confidential/ERC20Confidential.fsol
@@ -5,10 +5,10 @@ import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import { IERC165 } from "@openzeppelin/contracts/interfaces/IERC165.sol";
 import { ERC165 } from "@openzeppelin/contracts/utils/introspection/ERC165.sol";
-import { IFHERC20, IERC7984 } from "../interfaces/IFHERC20.sol";
-import { IERC20Confidential } from "../interfaces/IERC20Confidential.sol";
-import { ERC20ConfidentialIndicator } from "./ERC20ConfidentialIndicator.sol";
-import { ERC20ConfidentialCoreUpgradeable } from "./ERC20ConfidentialCoreUpgradeable.sol";
+import { IFHERC20, IERC7984 } from "../interfaces/IFHERC20.fsol";
+import { IERC20Confidential } from "../interfaces/IERC20Confidential.fsol";
+import { ERC20ConfidentialIndicator } from "./ERC20ConfidentialIndicator.fsol";
+import { ERC20ConfidentialCoreUpgradeable } from "./ERC20ConfidentialCoreUpgradeable.fsol";
 
 /**
  * @title ERC20Confidential
